identity-map/README.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: pattern
+title: Identity Map
+folder: identity-map
+permalink: /patterns/identity-map/
+categories: Object-Relational Behavioral
+language: en
+tags:
+- Performance
+---
+
+## Intent
+
+Ensures that each object gets loaded only once by keeping every loaded object in a map. 
+Looks up objects using the map when referring to them.
+
```

Comment:
Please add explanation section with a programmatic example. See how it's written in the other patterns.